pkg/kernelsyms/kernel_symbols.go

```diff
@@ -0,0 +1,86 @@
+package kernelsyms
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+type KernelSymbolTable struct {
+	symbolMap map[string]KernelSymbol
+}
+
+type KernelSymbol struct {
+	Name    string
+	Type    string
+	Address uint64
+	Owner   string
+}
+
+type HookedSyscallData struct {
+	SyscallName string
+	ModuleOwner string
+}
+
+const IoctlSyscallHook int = 65
+
+/*
+ * this package is to support a kernel-malware detection and especially rootkits.
```

Comment:
I think ill remove it because now we aren't support a lot of detections.
if in the future we will create a file or package for malware detection ill added it (: